DESCRIPTION

```diff
@@ -1,16 +1,22 @@
 Package: chipseq
 Title: chipseq: A package for analyzing chipseq data
 Version: 1.49.0
-Author: Deepayan Sarkar, Robert Gentleman, Michael Lawrence, Zizhen Yao
+Authors@R: c(
+        person("Deepayan", "Sarkar", role = "aut"),
+        person("Robert", "Gentleman", role = "aut"),
+        person("Michael", "Lawrence", role = "aut"),
+        person("Zizhen", "Yao", role = "aut"),
+        person("Oluwabukola", "Bamigbade", role = "ctb",
+               comment = "Converted Chipseq vignette from ChipseqFiles.Rnw to RMarkdown / HTML."),
```

Comment:
Let's clarify and shorten the comment by rewriting `Converted vignette from Sweave to R Markdown / HTML.`